Master-Game/Asset/Jay01.json

```diff
@@ -0,0 +1,104 @@
+﻿{
```

Comment:
Actually remove this file. It’s in the wrong place

Reply:
Just to follow up, we decided to put our maps in the Assets/Maps folder, so close.